docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '3'
+
+services:
+  db:
+    image: postgres
+    volumes:
+      - ./tmp/postgres_data:/var/lib/postgresql/data
+
+  redis:
+    image: redis
+
```

Comment:
It's a good practice add health-check in each docker service that is required by another.
Then, add something like:
`
healthcheck:
  test: ["CMD", "redis-cli", "ping"]
  interval: 30s
  timeout: 10s
  retries: 3 
`